packages/entities/entities-certificates/src/components/CACertificateConfigCard.vue

```diff
@@ -10,23 +10,30 @@
       @fetch:success="handleFetch"
       @loading="(val: boolean) => $emit('loading', val)"
     >
-      <template #cert="{ rowValue }">
+      <template #cert="slotProps">
```

Comment:
Question: why are we changing `{ rowValue }` and `{ row }` to `slotProps` and `getPropValue`? IMHO `{ rowValue }` is more straightforward.

Reply:
The problem is `slotProps` can contain either `rowValue` or `row` or both or neither and TS isn't smart enough to determine which properties will be available to it. In order to avoid TS squawking at us we needed to move the logic that pulls the prop out of the slot props out of the template.

Reply:
Got it. Thank you both!